gnmi_server/server_test.go

```diff
@@ -3828,14 +3828,14 @@ print('%s')
 	s := createServer(t, 8080)
 	go runServer(t, s)
 	defer s.s.Stop()
-	initFullConfigDb(t, sdcfg.GetDbDefaultNamespace())
-	initFullCountersDb(t, sdcfg.GetDbDefaultNamespace())
+	initFullConfigDb(t, sdcfg.GetDbDefaultInstance())
+	initFullCountersDb(t, sdcfg.GetDbDefaultInstance())
 
 	path, _ := os.Getwd()
 	path = filepath.Dir(path)
 
 	var cmd *exec.Cmd
-	cmd = exec.Command("bash", "-c", "cd "+path+" && "+"pytest")
+	cmd = exec.Command("bash", "-c", "cd "+path+" && "+"pytest -m 'not dpu'")
```

Comment:
> -m 'not dpu' [](http://example.com/codeflow?start=59&length=13)

Could you add some comment to explain the intention?

Reply:
updated